pkg/controller/clusterconfig/clusterconfig_controller.go

```diff
@@ -114,6 +115,17 @@ func (r *ReconcileClusterConfig) Reconcile(ctx context.Context, request reconcil
 				fmt.Sprintf("Failed to process network type live migration (%v). Use 'oc edit network.config.openshift.io cluster' to fix.", err))
 			return reconcile.Result{}, err
 		}
+		// Only allow live migration in managed clusters excluding HyperShift clusters
```

Comment:
Adding this check back because there are checks in both cluster config and oper config controllers. There is no separation  between the controllers currently - i.e. one does the checks. Continue that pattern.

Also, if we dont fail here, we're translating an unsupported config to oper config. Even though, its harmless, lets stop an unsupported config in both controllers.

Reply:
Makes sense. The operator config is also exposed to the user so agreed.